sys/compat/linuxkpi/common/include/linux/pci.h

```diff
@@ -311,7 +311,7 @@ struct pci_mmio_region {
 };
 
 struct pci_dev {
-	struct device		dev;
+	struct device		dev __subobject_use_container_bounds;
```

Comment:
__subobject_member_used_for_c_inheritance maybe? Depends if it's embedding or inheriting.

Reply:
Agree this is inheritance case.